core/src/main/java/io/ddf/etl/IHandleTransformations.java

```diff
@@ -44,12 +44,29 @@ DDF transformPython(String[] transformFunctions, String[] functionNames,
    *                             with newColumnNames.
    * @param selectedColumns list of column names to be included in the result DDF.
    *                        If null or empty, all existing columns will be included.
-   * @return current DDF if it is mutable, or a new DDF otherwise.
+   * @return new DDF
    * @throws DDFException
    */
   DDF transformUDFWithNames(String[] newColumnNames, String[] transformExpressions,
```

Comment:
One question is: why we didn't do case-insensitive check here? @Huandao0812 @hai-adatao This function only did case sensitive check here.